languages/de_de/Settings/Colors.php

```diff
@@ -4,7 +4,7 @@
  * @package YetiForce.Languages
  * @copyright YetiForce Sp. z o.o.
  * @license YetiForce Public License 3.0 (licenses/LicenseEN.txt or yetiforce.com)
- * @author Sławomir Kłos <s.klos@yetiforce.com>
```

Comment:
The author never removes just add another.

Reply:
Okay I agree, but he has not done a translation yet and was mentioned at the initial provision of the file, otherwise in no single German translation. I assumed this was a copy and paster error or do you see it differently?

Reply:
This person created the file so also is the author

Reply:
I thought KatarzynaUlichnowska copied the file from the english translation. But if not a problem I will add the change again.